ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -186,7 +186,7 @@ tests:
       SOURCE_BRANCH: master
     workflow: windows-ci-fastforward
 - always_run: false
-  as: nutanix-e2e-operator
+  as: nutanix-e2e-operator-win
```

Comment:
Why do you need to rename this? This test can only be run from the WMCO repo and I don't see a need to add `-win` to this. Sure there is another test named `e2e-nutanix-operator` but when looking at prow the full context will make it obvious. So this change is not needed.

Reply:
Example: [pull-ci-openshift-windows-machine-config-operator-master-nutanix-e2e-operator](https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/pull-ci-openshift-windows-machine-config-operator-master-nutanix-e2e-operator)

Reply:
Okay, got it. Removed the rename commit.